modules/ivc_champva/app/controllers/ivc_champva/v1/uploads_controller.rb

```diff
@@ -23,11 +23,12 @@ def submit
 
           status, error_message = FileUploader.new(form_id, metadata, file_paths, attachment_ids, true).handle_uploads
 
-          render json: build_json(Array(status), error_message)
+          render json: build_json(Array(status), error_message), status:
         rescue => e
           puts 'An unknown error occurred while uploading document(s).'
           Rails.logger.error "Error: #{e.message}"
           Rails.logger.error e.backtrace.join("\n")
+          render json: build_json([500, error_message], error_message), status: :internal_server_error
```

Comment:
This is the main contribution - an explicit error response when we hit this rescue block so the front end knows to stop what it's doing and show an error.